app/ssttpeligibility/EligibilityServiceConfig.scala

```diff
@@ -20,17 +20,18 @@ import javax.inject.Inject
 import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
 final case class EligibilityServiceConfig(insignificantDebtUpperLimit:                      Int,
-                                          maximumDebtForSelfServe:                          Int,
                                           numberOfDaysAfterDueDateForDebtToBeConsideredOld: Int,
                                           returnHistoryYearsRequired:                       Int,
                                           taxYearEndMonthOfYear:                            Int,
                                           taxYearEndDayOfMonth:                             Int) {
+
   @Inject def this(servicesConfig: ServicesConfig) = this(
     insignificantDebtUpperLimit                      = servicesConfig.getInt("eligibility.insignificantDebtUpperLimit"),
-    maximumDebtForSelfServe                          = servicesConfig.getInt("eligibility.maximumDebtForSelfServe"),
```

Comment:
Possible debugging - was this meant to be removed by this ticket?

Reply:
nope